src/people/main/FocusView.tsx

```diff
@@ -282,19 +283,33 @@ function FocusedView(props: FocusViewProps) {
       if (window.location.href.includes('wanted')) {
         await main.getPersonCreatedBounties({}, info.pubkey);
       }
-    } catch (e) {
-      console.log('e', e);
-    }
 
-    setIsEditButtonDisable(false);
-    setSubmiting(false);
-    if (props?.onSuccess) props.onSuccess();
+      setIsEditButtonDisable(false);
+      setSubmiting(false);
+      if (props?.onSuccess) props.onSuccess();
+
+      if (notEdit === true) {
+        setLoading(false);
+      }
+      if (
+        ui?.meInfo?.hasOwnProperty('url') &&
+        !isNotHttps(ui?.meInfo?.url) &&
+        props?.ReCallBounties
+      )
+        props?.ReCallBounties();
+    } catch {
+      setToasts([
+        {
+          id: '1',
+          title: 'Something went wrong! Unable to create bounty'
+        }
+      ]);
 
-    if (notEdit === true) {
-      setLoading(false);
+      setTimeout(() => {
+        handleFormClose();
+      }, 6000);
```

Comment:
not sure why we are using `setTimeout` I don't think we should close out of the bounty creation window if creating the bounty fails